src/main/kotlin/no/nav/familie/tilbake/config/FeatureToggleConfig.kt

```diff
@@ -16,16 +16,6 @@ class FeatureToggleConfig(
     fun strategies(): List<Strategy> = listOf(ByClusterStrategy(clusterName))
 
     companion object {
-        const val KAN_OPPRETTE_BEH_MED_EKSTERNID_SOM_HAR_AVSLUTTET_TBK =
```

Comment:
Den her tror jeg man fort kan få bruk for med (u)jevne mellomrom. Sist gang den ble brukt var 2. desember, men det kan fort hende at vi får bruk for den igjen:

https://teamfamilie-unleash-web.iap.nav.cloud.nais.io/projects/default/features/familie-tilbake.beh.kanopprettes.eksternid.avsluttet.tilbakekreving/logs

Reply:
Ser ut som den blir "påskrudd" by default, isåfall er det kanskje greit? Kanskje høre med BAKS / @bragejahren hva de tenker?

Reply:
Den er av nå som gjør at valideringen er på. Innså at jeg har lest datoen i unleash feil fordi den er på amerikansk format 🤦

Reply:
Ja, denne har jeg brukt av og til. Det er langt mellom hver gang, men som beskrivelsen av toggelen sier så er det et scenario hvor det kan skje at vi mottar kravgrunnlag i flere omganger og man rekker å behandle tilbakekrevingen basert på det første kravgrunnlaget før resten av kravgrunnlaget kommer. Er nok litt edge, men det skjer slik løsningen er nå. Usikker på om det er noe vi kan gjøre for å håndtere det på en annen måte 🤔

Reply:
Da har jeg lagt tilbake feature toggle, fikset testene for den og lagt til en test for tilfellene hvor den er slått av